nssp/delphi_nssp/patch.py

```diff
@@ -0,0 +1,139 @@
+"""
+This module is used for patching data in the delphi_nssp package.
+
+To use this module, you need to turn on the custom_run flag
+and specify the range of issue dates in params.json, like so:
+
+{
+  "common": {
+    "custom_run": true,
+    ...
+  },
+  "validation": {
+    ...
+  },
+  "patch": {
+    "patch_dir": "/Users/minhkhuele/Desktop/delphi/covidcast-indicators/nssp/AprilPatch",
+    "start_issue": "2024-04-20",
+    "end_issue": "2024-04-21",
+    "source_dir": "/Users/minhkhuele/Desktop/delphi/covidcast-indicators/nssp/source_data"
```

Comment:
question: what is the `source_dir` used for? Isn't NSSP data fetched from an API?

Reply:
nssp data is fetched from an API when it is run normally. The API does not have versioning, so we keep a daily stash of what the api returns every day on a separate server, which is used here as source data for issue dates in patching.